lib/ansible/modules/cloud/amazon/rds_v2.py

```diff
@@ -0,0 +1,1148 @@
+        db_parameter_group_name=dict(type='str', required=False),
+        backup_retention_period=dict(type=int, required=False),
+        preferred_backup_window=dict(type='str', required=False),
+        copy_tags_to_snapshot=dict(type='bool', required=False),
+        monitoring_interval=dict(type='int', required=False),
+        port=dict(type='int', required=False),
+        multi_az=dict(type='bool', default=False),
+        engine_version=dict(type='str', required=False),
+        auto_minor_version_upgrade=dict(type='bool', required=False),
+        allow_major_version_upgrade=dict(type='bool', required=False),
+        license_model=dict(type='str', choices=['license-included', 'bring-your-own-license', 'general-public-license', 'postgresql-license'], required=False),
+        iops=dict(type='int', required=False),
+        option_group_name=dict(type='str', required=False),
+        publicly_accessible=dict(type='bool', required=False),
+        character_set_name = dict(type='str', required=False),
+        tags=dict(type='dict', default=None, required=False),
+        storage_encrypted=dict(type='bool', required=False),
+        kms_key_id=dict(type='str', required=False),
+        wait=dict(type='bool', default=False),
+        wait_timeout=dict(type='int', default=320, required=False),
+        new_db_instance_identifier=dict(type='str', required=False),
+        storage_type=dict(type='str', required=False),
+        db_snapshot_identifier=dict(type='str', required=False),
+        skip_final_snapshot=dict(type='bool', default=False),
+        final_db_snapshot_identifier=dict(type='str', required=False),
+        force_failover=dict(type='bool', default=False),
+        db_cluster_identifier=dict(type='str', required=False),
+        source_db_instance_identifier=dict(type='str', required=False)
+        )
+    )
+    module = AnsibleModule(argument_spec=argument_spec)
+
+    if not HAS_BOTO3:
+        module.fail_json(msg='json and boto3 is required.')
+
+    try:
+        region, ec2_url, aws_connect_kwargs = get_aws_connection_info(module, boto3=True)
+        client = boto3_conn(module, conn_type='client', resource='rds', region=region, endpoint=ec2_url, **aws_connect_kwargs)
+    except botocore.exceptions.NoCredentialsError:
+        e = get_exception()
+        module.fail_json(msg="Can't authorize connection - "+str(e))
+
+    invocations = {
+            'create': create_db_instance,
+            'modify': modify_db_instance,
+            'snapshot': snapshot_db_instance,
+            'delete': delete_db_instance_or_snapshot,
+            'restore': restore_db_instance_from_snapshot,
+            'reboot': reboot_db_instance,
+            'replicate': replicate_db_instance,
+            'promote': promote_db_instance
+    }        
+
+    changed, results = invocations[module.params.get('command')](client, module)
+
+    module.exit_json(changed=changed, rds=results)
```

Comment:
Use the ec2 helper function `camel_dict_to_snake_dict` to ensure the results are returned in snake_case format